src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(callback, value) {
+      callback(value);
+
+      return this;
+    },
+
+    add(value) {
+      calculator.result += value;
+    },
+
+    subtract(value) {
+      calculator.result -= value;
+    },
+
+    multiply(value) {
+      calculator.result *= value;
+    },
+
+    divide(value) {
+      calculator.result /= value;
+    },
+
+    reset() {
+      this.result = 0;
```

Comment:
why do you use `calculator.result` in the others methos, but `this.result` here?

Reply:
I had to rewrite code and I forgot to rewrite it there)